runtime/kusama/src/lib.rs

```diff
@@ -1428,8 +1446,9 @@ construct_runtime! {
 		// Preimage registrar.
 		Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 32,
 
-		// Bounties module.
+		// Bounties modules.
 		Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 35,
+		ChildBounties: pallet_child_bounties = 40,
```

Comment:
We just pick a number that is not used yet?

Reply:
Yes, each pallet needs a unique index for encoding into call enum